packages/odyssey-react/src/components/Radio/RadioButton.module.scss

```diff
@@ -134,40 +104,43 @@
 .label {
   display: block;
   position: relative;
-  margin-block-end: $spacing-xs;
-  padding-inline-start: 1em + $spacing-s-em;
-  font-size: $size-body-base;
-  font-weight: 400;
+  margin-block-end: var(--MarginBlockEnd);
+  padding-inline-start: calc(
+    var(--CircleSize) + var(--PaddingLabelInlineStart)
```

Comment:
oh goodness names are fun. `var(--PaddingLabelInlineStart)`

We're missing a clear standard for this type of complicated example. See my comment above for more challenging use cases.

Some of @edburyenegren-okta 's work was adding a "part" prefix so in this case:
`var(--LabelSpacePaddingInlineStart)` I'm not sure I have a strong opinion here until we can sync up on a standard during office hours :soon:

Reply:
@andrewberg-okta @edburyenegren-okta for what it is worth I decided to try and experiment with following the ordering of name parts in our current name system more strictly in my draft `withTheme: Text` PR. 😅 So far I feel this works better for clarity because the API is more uniform and predictable. Lets talk more :soon: